src/controllers/profile.js

```diff
@@ -0,0 +1,110 @@
+import { config } from 'dotenv';
+import cloudinary from 'cloudinary';
+import { User } from '../sequelize/models';
+
+config();
+cloudinary.config({
```

Comment:
This cloudinary config would be better if you put it in config folder. And rename the name of this controller to `profileController`

Reply:
on it. thanks